### DISCUSSION TITLE
Synology Postgres DB move to Volume 2

### DISCUSSION THREAD

**Discussion by ctrlaltdelete007** [007revad/Synology_app_mover#53] [Ideas]
Hi Dave,
do you think its possible to move the postgres db permanently to volume 2 (NVME-SSD)
https://www.reddit.com/r/synology/comments/14t0c37/move_postgres_db_to_another_volume/
Cheers
Sven

**Comment by ctrlaltdelete007** [comment]
Maybe with the following commands?

Stop the pgsql:
"systemctl stop pgsql-adapter.service"

Then create the folder: 
"mkdir /volume2/@database/pgsql"

Move with:
"mv /volume1/@database/pgsql /volume2/@database/pgsql"

Change the rights:
"chown postgres.postgres /volume2/@database/pgsql"

Then you can start the pgsql:
"systemctl start pgsql-adapter.service"

And check if it is ok:
"systemctl status pgsql-adapter.service"

and redo the symlink (/var/services/pgsql) so that it points to the SSD volume
("mount --bind /volume1/@database/pgsql /volume2/@database/pgsql")
"vi /var/services/pgsql"
"mount --bind /volume1/@database/pgsql /volume2/@database/pgsq"

**Comment by 007revad** [comment]
That would work, but you'd want to edit the symlink before starting pgsql. To do that I'd use:
```
rm /var/services/pgsql
ln -s /volume2/@database/pgsql /var/services/pgsql
```

Thanks for the donation.

**Comment by ctrlaltdelete007** [comment]
ok, it worked, but the synology photo index was gone.
Do you know a way to migrate the photo index, albums, face recognition?

**Comment by 007revad** [comment]
I thought synology photos stores what it needs, like metadata, in a SYNOINDEX_MEDIA_INFO file in the `@eaDir` in each folder like: 
`/volume1/homes/Dave/Photos/PhotoLibrary/2012/01/@eaDir/15526.jpg/SYNOINDEX_MEDIA_INFO`

The there are .DB files in `/var/packages/SynologyPhotos/target/indexdb` but they aren't mysql files. Maybe they are packed. But they seem to only be language specific files for the ui. All of my files in that folder have the date from when Synology created the package.

I haven't found where synology photos stores it's index yet. But I have found this in `/var/packages/SynologyPhotos/conf/resource`
```
    "media-index": {
        "info-relpath": "usr/index/INFO",
        "plugin-relpath": "usr/index/libsynofoto-index-plugin.so"
    },
```
**EDIT** Nothing useful found in `/var/packages/SynologyPhotos/target/usr/index`

I did see "migrate" in another file but I believe it's for migrating from photostation to synology photos.

**Comment by 007revad** [comment]
Is it the index from the shared space that is missing?

If you've moved the shared space folder to volume 2 try this:
1. Go to Synology Photos > Settings > Shared Space.
2. Disable the Shared Space.
3. Then go back in and Enable Shared Space.
